File: dongyimaimangerweb/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {

    this.delete=function (ids) {
        return $http.get('../brand/delete.do?ids='+ids);
    }

    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }

    this.save=function (methodName,entity) {
        return $http.post('../brand/'+methodName+'.do',entity);
    }

    this.search=function (pageNum,pageSize,searchEntity) {
        return $http.post('../brand/search.do?pageNum='+pageNum+"&pageSize="+pageSize,searchEntity);
    }

    this.findAll = function () {
        return $http.get('../brand/selectOptionList.do');
    }

});

File: dongyimaisellergoodsinterface/src/main/java/com/offcn/sellergoods/service/BrandService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbBrand;

import java.util.List;

public interface BrandService {

    public List<TbBrand> findAll();

    public PageResult findPages(int pageNum, int pageSize);

    public void save(TbBrand brand);

    public TbBrand findOne(long id);

    public void update(TbBrand brand);

    public void delete(long[] ids);

    public PageResult search(TbBrand brand,int pageNum,int pageSize);

    public List<TbBrand> selectOptionList();
}